website/docs/addon_deadline_admin.md

```diff
@@ -16,107 +16,182 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 
-It submits a dependent AYON Job to publish the rendered images.
-- Find the setting at: `ayon+settings://deadline/publish/ProcessSubmittedJobOnFarm`
+If you have the `PATH` configured in your application addon as mentioned in  [How to submit houdini patch version to deadline?](addon_houdini_admin.md#how-to-publish-lookdev-from-houdini-), then you'd be able to only use `husk.exe` instead of the full path.
+![](assets/deadline/husk_standalone_plugin.png)
+:::
 
-![](assets/deadline/process_submitted_job_on_farm.png)
+## Addon Settings
 
-1. **Department**
-2. **Pool**
-3. **Group**
-4. **Priority**
-5. **Skip integration of representation with ext:** list of extensions that shouldn't be published.
-6. **List of family names to transfer to generated instances (AOVs for example):** Ensure the AOV instances have these families if present.
-7. **Reviewable products filter:** Add review for specific aov names.
-![](assets/deadline/reviewable_products_filter.png)
-   1. **AOV Filter**
-       1. **host_name:** Pipeline integration name supplied by an AYON addon.
-       2. **AOV Regex:** AOV regex patterns for AOV filters.
-       3. **+ icon:** Add more AOV regexes 
-   2. **+ icon:** Add more filters
 
-### Add AYON server to farm job
+### System Deadline Webservice Info
+> Setting Location: `ayon+settings://deadline/deadline_urls`
+
+![Webservice url](assets/deadline/webserver_config.png)
+
+
+- `Url` - is IP or host name including port (example of locally running DL `http://127.0.0.1:8081`)
+- `Require authentication` - DL webservice could be protected by username/password (set in Deadline Monitor in `Tools > Configure Repository Options > Web Service Settings > Require Authentication`)
+- `Don't verify SSL` - if your Dl webservice is using SSL (eg. on https://) self-signed certificates might trigger an error. Disable verification of certificate here.
+- `Default user name` - if `Require authentication` is enabled, this and next field provides possibility to fill single credentials for all artists publishing to Deadline. 
+  
+:::info
+If every machine should have separate credentials, admin need to provide them in `Site Settings` on `Studio Settings` page.
+
+![](assets/deadline/studio_settings.png)
+:::
 
+## Addon Publish Plugins
+
+### Collect Job Info
+> Setting Location: `ayon+settings://deadline/publish/CollectJobInfo`
+
+![](assets/deadline/collect_job_info.png)
+
+
+#### Profiles
+<div class="row markdown">
+<div class="col">
+
+- Host names
+- Task Types
+- Task names
+- Frames per Task
+- Priority
+- Group
+- Limit Groups
+- Primary Pool
+- Secondary Pool
+- Machine Limit
+- Machine List
+- Machine List is a Deny
+- Number of concurrent tasks
+- Department
+- Use GPU
+- Delay Job
+- Use Published Scene
+- Use Asset dependencies
+- Workfile Dependency
+- Multiprocess
+- Allowed environment keys
+- Search & replace in environment values
+- Additional JobInfo data
+- Additional PluginInfo data
+- Exposed Overrides
+- **+** button
```

Comment:
What's the idea here? Still needs more info, or? Feels a bit useless 🗡️

Reply:
needs more info.

Reply:
should be fixed here https://github.com/ynput/ayon-documentation/pull/301/commits/cb341a3f25116212bdb21eac221d497f4df8c06a (But not all of them unfortunately)